docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -472,12 +472,15 @@ query brands {
 
 ## Complexity limits
 
-The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. Queries that exceed the complexity score will receive an error response:
+The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. The API returns the query complexity as an integer in the HTTP response header `x-bc-graphql-complexity` when you send a valid GraphQL request.
+
+If a query's complexity score exceeds the complexity limit, you will receive the following error response:

```

Comment:
```suggestion
If a query's complexity score exceeds the complexity limit, you will receive an error response similar to the following:

```

the score will be different each time. also the text might change a little in the future.